server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -213,22 +223,52 @@ private String getVideoId(String response) {
    * @param comments The array to be condensed.
    * @return A properly formatted String
    */
-  private String convertToString(ArrayList<String> comments) {
+  private String convertToString(ArrayList<Comment> comments) {
     StringBuilder res = new StringBuilder();
 
-    for (String comment : comments) {
-      comment = comment.replace("\"", "");
+    for (Comment comment : comments) {
+      String commentText = comment.text;
+
+      commentText = commentText.replace("\"", "");
       // Make sure each comment is treated as its own sentence
       // Not sure char datatype works with regex so used String
-      String lastCharacter = comment.substring(comment.length() - 1);
+      String lastCharacter = commentText.substring(commentText.length() - 1);
       if (!lastCharacter.matches("\\.|!|\\?")) {
-        comment += ". ";
+        commentText += ". ";
       } else {
-        comment += " ";
+        commentText += " ";
       }
-      res.append(comment);
+      res.append(commentText);
     }
 
     return res.toString();
   }
+
+  /**
+   * Filters out a comment string so that it's readable and doesn't break the frontend.
+   *
+   * @param comment The comment to be filtered.
+   * @return A properly formatted comment
+   */
+  private String filterComment(String comment) {
+    String filteredComment = comment;
+
+    // Removes 4683 different emojis and symbols so that NL and Perspective don't crash
+    // \u00a9 : copyright character
+    // \u00ae : registered sign
+    // \u2000-\u3300 : char code ranges 8192 to 13056
```

Comment:
This comment doesn't tell me anything that the code itself doesn't. What does this range represent?